terraform/app/variables.tf

```
@@ -256,6 +256,12 @@ variable "minimum_good_job_replicas" {
   description = "Minimum amount of allowed replicas for good-job service"
 }
 
+variable "max_acu_capacity" {
```

```suggestion
variable "max_database_acu_capacity" {
```

Might be worth just making clear that this refers to the database.